### PR TITLE
Update dataminer.services change log 38772 

### PR DESCRIPTION
**Pull request by AlexanderVK-SLC**
Systems with remote access disabled are now shown on dataminer.services

### CODE REVIEW

**Comment by AlexanderVK-SLC** [comment]
Wait for the feature to be merged & released before merging this docs update.

**Review by AlexanderVK-SLC — COMMENTED** [review]
Wait for the feature to be merged and released before merging this docs update.

File: release-notes/Cloud/DCP_change_log.md
Comment (MariekeGO) [inline release-notes/Cloud/DCP_change_log.md:15]:
```suggestion
DataMiner Systems are now shown on dataminer.services even if remote access is disabled for them. However, the app buttons for such systems will be disabled where necessary. A link to the documentation is provided in the UI for more information about how to change the remote access settings.
```

**Comment by AlexanderVK-SLC** [comment]
Can now be merged, the change is live.

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**
